.github/workflows/main.yml

```diff
@@ -9,6 +9,10 @@ on:
       - '**.md'
   workflow_dispatch:
 
+concurrency:
+  cancel-in-progress: true
```

Comment:
This will cancel in-progress builds on the same branch when new commits are pushed. I generally like this because it avoids wasting resources when someone pushes two commits in a short time. But I don't want to force it, so let me know if you would prefer to remove that.

Reply:
I like this idea, @JeanChristopheMorinPerso, and I'd have no objection to including this change as part of the PR.